### DISCUSSION TITLE
All Exercise Minutes by date range.

### DISCUSSION THREAD

**Discussion by write2lloyd** [agencyenterprise/react-native-health#133] [Q&A]
Hi,
Thanks for creating this library. We are building an App which uses Google Fit on Android and this lib for iOS to display Steps, Sleep and Exercise mins.

I want to get all the activity or exercise minutes  by date range in minutes

Not sure how to go about it. We have the `getSamples` method but that can only pick one out of the many activities like 'Walking" or "Running" etc..

Would appreciate any help or steer in the right direction.

Thanks in advance,
Lloyd

**Comment by ohnomalone** [comment]
This is something I am trying to figure out as well! I want to get total minutes OR all of the activities the user has participated in. From there I can create the minute total. 

Thank you,
Matt
